state/azure/cosmosdb/storedprocedure.go

```diff
@@ -0,0 +1,115 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package cosmosdb
+
+const spDefinition string = `// upserts - an array of objects to upsert
+// deletes - an array of objects to delete
+
+function dapr_multi(upserts, deletes) {
+    var context = getContext();
+    var container = context.getCollection();
+    var response = context.getResponse();
+    
+    if (typeof upserts === "string") {
+        throw new Error("first arg is a string, expected array of objects");
+    }
+
+    if (typeof deletes === "string") {
+        throw new Error("second arg is a string, expected array of objects");
+    }
+
+    // create the query string used to look up deletes
+    var query = "select * from n where n.id = ";
```

Comment:
Can we use `IN` query keyword? - https://docs.microsoft.com/en-us/azure/cosmos-db/sql-query-keywords#in

it will reduce the length of query. Do we know the length of single query?

Reply:
`IN` would be applicable here and regarding query string length you would safe all the " or = " string segments.

Reply:
https://docs.microsoft.com/en-us/azure/cosmos-db/concepts-limits#sql-query-limits 256KB ... I think it should be a problem in normal case.

Reply:
Modified it to use `in`.

Reply:
*it should not be a problem :)